src/settings.rs

```diff
@@ -176,6 +176,7 @@ impl Settings {
         s.set_default("tokenserver.fxa_email_domain", "test.com")?;
         s.set_default("tokenserver.fxa_metrics_hash_secret", "secret")?;
         s.set_default("tokenserver.test_mode_enabled", false)?;
+        s.set_default("tokenserver.node_type", "spanner")?;
```

Comment:
It's sufficient to use a setting to set the node type, since a given instance of Tokenserver will only ever be responsible for Spanner nodes **or** MySQL nodes, but never both